Dockerfile

```diff
@@ -33,8 +33,7 @@ LABEL vendor=Sonatype \
 USER root
 
 # For testing
-RUN microdnf update \
-&& microdnf install procps
+RUN microdnf install procps
```

Comment:
is it ok to lose the update? how up-to-date is the base image? how else can the problem be resolved while still updating all the software?

Reply:
the "for testing" comment leads me to believe neither action is essential.

Reply:
there are customers that hate docker images with old components. but this is our general problem with keeping docker images up-to-date. we don't have to solve this as part of this PR. @mpuglin 
what confuses me is why the update fails to update the rpm-libs. but i'm not much of a redhat person.

Reply:
if I `dnf update --allowerasing -y`, it'll update 281 packages (today). 🤔

Reply:
^^ by using the `--allowerasing` option, I can get it to finish the build that way too, but I again, I don't know enough context to know what's best.

Reply:
I think it's okay to skip the update , if someone wants to update they can do it by executing `RUN microdnf update`, also this is what's blocking release 101 right now, so for me, it's okay to merge and maybe create a follow-up ticket?

Reply:
i don't know enough about DNF to be sure what's the best way. -> follow up ticket?

Reply:
i think the previous way it built did yum install of procps, but didn't do the update, so skipping the update now would be consistent, i think.